src/index.js

```diff
@@ -154,7 +158,8 @@ class ReactPhoneInput extends React.Component {
       queryString: '',
       showDropdown: false,
       freezeSelection: false,
-      debouncedQueryStingSearcher: debounce(this.searchCountry, 100)
+      debouncedQueryStingSearcher: debounce(this.searchCountry, 100),
+      countryCodeEditable: props.countryCodeEditable
```

Comment:
Please delete this and use prop directly from the props instead of appending it to the component state.